File: Euclid's Algorithm/euclid.py
def main():
	print("Enter Two Number To Find GCD")
	m = int(input("Enter a Number:")) 
	n = int(input("Enter a Number:")) 
	if m<n:
		m,n = n,m

	while n!=0:	
		r = m%n
		m = n 
		n = r 

	print("GCD",m)
	



if __name__ == '__main__':
	main()

File: Euclid's Algorithm/euclid.c
#include<stdio.h>
#include<conio.h>

int main()
{    
    int m,n,r,temp;
    printf("To Find GCD");
    printf("\n Enter The Num");
    scanf("%d",&m);
    printf("\n Enter The Num");
    scanf("%d",&n);
    if(m<n)
    {
        temp = m;
        m = n;
        n =temp;
    }
    
    while(0==0)
    {    
        if(n==0)
           { printf("\n GCD is %d ",m);
            break;}
        else
        {
            r=m%n;
            m=n;
            n=r;
        }
    }
}